### PR TITLE
[TFF-191] Added view for model and all its items

### PR DESCRIPTION
**Pull request by tondef001**
### Summary

There are now model pages that contain the model component and the item components for each child item.

### Checklist

- [x] Have you added unit and functional tests as appropriate?
- [x] Did you update/add documentation for new methods or changed functionality?
- [x] Have you merged the target branch down into this one?

### How to Review

1. There are currently 2 ways to view the page. Either enter a model address into the omnibar, or go to the view all items page and click the "View model" button on an item.
2. Bam. You see a page with a model and all items of that model
3. Run tests
4. You can also enter an item address into the omnibar to see that item. Do that too
5. Have fun

### Links

- [TFF-191](https://msoese.atlassian.net/browse/TFF-191)
- TheFourFifths/consus/pull/81

### CODE REVIEW

File: src/views/components/omnibar.jsx [hunk 1]
```diff
@@ -13,16 +15,29 @@ export default class Omnibar extends React.Component {
...
                 });
-                OmnibarController.getStudent(e.target.value);
-            } else {
-                this.setState({
-                    query: e.target.value
-                });
+                if(result.type === 'model') {
+                    ModelController.getModelAndItems(e.target.value);
+                } else if (result.type === 'item') {
+                    OmnibarController.displayItem(e.target.value);
```
Comment (mattmahn) [inline src/views/components/omnibar.jsx]: This is a bit smelly; why is it `ModelController` for showing a model & items, and `OmnibarController` for displaying an item.

File: src/views/components/omnibar.jsx [hunk 2]
```diff
@@ -13,16 +15,29 @@ export default class Omnibar extends React.Component {
...
+                }
+            } catch (f) {
+                if (e.target.value.length === 6 && regexOnlyNums.test(e.target.value)) {
```
Comment (mattmahn) [inline src/views/components/omnibar.jsx]: Remember in SE1021 when you were told that Exceptions are only for exceptional behavior, not meant for typical program flow? Entering a student ID in the omnibar isn't that exceptional.

My only solution is:
```
if (e.target.value.length === 6 && regexOnlyNums.test(e.target.value)) {
  // show student
} else {
  // do all the model/item stuff
}
```

File: test/unit/lib/api-client.js [hunk 3]
```diff
@@ -244,6 +245,32 @@ describe('API Client', () => {
         });
     });
 
+    it('getModelAndItems', () => {
+        let response = {
+            status: 'success',
+            data: {
+                model: 'm8y7nEtAe',
```
Comment (mattmahn) [inline test/unit/lib/api-client.js]: NACK: This doesn't match what the server responds with. https://github.com/TheFourFifths/consus/blob/25fcae2936a6795228acb40312ef1cad1f2d3966/docs/REST.md#get-apimodelchildren

File: src/views/components/omnibar.jsx (hunk 1 above)
Comment (tondef001) [inline src/views/components/omnibar.jsx]: I could add a getModelAndItems to omnibar controller, but it would just be duplicate code.  Seems pointless

**Pull request merged**